System.Exception: IsTrue encountered NullReferenceException, expression was:

l.ToUpperInvariant() == "Oops"
. .              .   __
. \______ _______/   |
.        |           |
|        |           (threw NullReferenceException)
|        (threw NullReferenceException)
null


 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at lambda_method(Closure )
   at PowerAssert.PAssert.IsTrue(Expression`1 expression, CallerLocation location) in ...\PAssert.cs
   --- End of inner exception stack trace ---
   at PowerAssert.PAssert.IsTrue(Expression`1 expression, CallerLocation location) in ...\PAssert.cs
   at PowerAssertTests.Approvals.EndToEndTest.ApproveException(Expression`1 func) in ...\EndToEndTest.cs